src/routes/reviewRouter.ts

```diff
@@ -21,21 +21,15 @@ export class ReviewRouter implements RouterInterface {
 	public register = (): Router => {
 		const router: Router = Router();
 
-		router.use(this._requestAuth.checkLoggedIn);
-
 		router.get('/',
-			this._requestAuth.checkIsVolunteer,
-			this._reviewController.reviewPage.bind(this._reviewController));
+			this._requestAuth.withAuthMiddleware(this, this._reviewController.reviewPage));
 
 		router.get('/next',
-			this._requestAuth.checkIsVolunteer,
-			this._reviewController.nextReview.bind(this._reviewController));
+			this._requestAuth.withAuthMiddleware(this, this._reviewController.nextReview));
 
 		router.post(
 			'/submit',
-			this._requestAuth.checkLoggedIn,
-			this._requestAuth.checkIsVolunteer,
```

Comment:
I'm taking it that this `checkIsVolunteer` check is done by the auth system directly now, rather than the applications system?

Reply:
Yeah, the new `withAuthMiddleware` will call a method on `hs_auth` and check if the user has access to the route based on the URI